ts/features/bonus/bpd/store/reducers/details/activation/technicalAccount.ts

```diff
@@ -0,0 +1,52 @@
+import { getType } from "typesafe-actions";
+import {
+  remoteError,
+  remoteLoading,
+  remoteReady,
+  remoteUndefined,
+  RemoteValue
+} from "../../../../model/RemoteValue";
+import { Action } from "../../../../../../../store/actions/types";
+import { bpdDeleteUserFromProgram } from "../../../actions/onboarding";
+import { bpdLoadActivationStatus } from "../../../actions/details";
+import { bpdUpsertIban } from "../../../actions/iban";
+import { IbanStatus } from "../../../../saga/networking/patchCitizenIban";
+
+/**
+ * This reducer keeps the latest valid technicalAccount (technical IBAN) for the user.
+ * This value can change when:
+ * - The application load the first time the value
+ * - The user choose to edit / add a new paymentInstrument and the operation is completed with success.
+ * - The user choose to unsubscribe from the cashback
+ *
+ * If the action `bpdLoadActivationStatus.success` is dispatched after a call to citizen v1 API, so the
+ * the technical account data is not available, the state will be remoteUndefined.
+ * @param state
+ * @param action
+ */
+const technicalAccountReducer = (
+  state: RemoteValue<string | undefined, Error> = remoteUndefined,
+  action: Action
+): RemoteValue<string | undefined, Error> => {
+  switch (action.type) {
+    case getType(bpdDeleteUserFromProgram.success):
+      return remoteUndefined;
+    case getType(bpdLoadActivationStatus.request):
+      return remoteLoading;
+    case getType(bpdLoadActivationStatus.success):
+      return action.payload.technicalAccount
+        ? remoteReady(action.payload.technicalAccount)
+        : remoteUndefined;
```

Comment:
```suggestion
      return remoteReady(action.payload.technicalAccount)
```
I think is better distinguish between a date loaded but undefined `remoteReady(undefined)` and a date that isn't still loaded `remoteUndefined`, because using the same value (`remoteUndefined`) to represents two different states could be lead to some problems if some component relies on the remote state to command, for example, a new loading. What you think=

Reply:
Yes, you're right. 
Updated in [973f6af](https://github.com/pagopa/io-app/pull/2969/commits/973f6afc4413a46d4d1f614cff4b1989a3d82710)